### PR TITLE
Add option to pass in custom stun urls

### PR DESCRIPTION
**Pull request by nevenduranec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,55 @@
...
 
 A library to detect your local IP address via WebRTC on the web page.
 
 ## Live demo
-https://vitalets.github.io/webrtc-ips/demo/
 
+https://vitalets.github.io/webrtc-ips/demo/
 
 ## Installation
+
```
Comment (vitalets) [inline README.md]: Please keep the original codestyle (single quotes, no extra spaces).

File: README.md [hunk 2]
```diff
@@ -1,45 +1,55 @@
...
 ```js
-import {getIPs, getIPv4, getIPv6} from 'webrtc-ips';
+import { getIPs, getIPv4, getIPv6 } from "webrtc-ips";
 
 const ips = await getIPs();
 // => [{address: '95.108.174.12', v6: false}, {address: '2a02:6b8::408:5830:47a6:d045:a9ac', v6: true}]
 
+// You can pass in your custom stun server urls
+const ips = await getIPs({ urls: "stun:stun.stunprotocol.org:3478" });
```
Comment (vitalets) [inline README.md:25]: I suggest to name it more precisely `stunUrls`.

**Review by vitalets — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (nevenduranec) [reply README.md:25]: This gets passed as is to https://github.com/vitalets/webrtc-ips/pull/1/files/1a57ef7b60862793701bf97f6056224860e56821#diff-0ea7f12adbcda08b8aecde80363ce603R31 if I rename it I would need to modify the final object to match the [spec](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceServer/urls). 

So the idea is that this needs to match the spec object without us modifying it.

Comment (vitalets) [reply README.md:25]: Oh, agree. In that case better to pass as is, for sure 👍

**Review by vitalets — APPROVED** [review]

**Comment by vitalets** [comment]
Thank you!

**Pull request merged**